Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/git_unittest.py

```diff
@@ -342,10 +342,11 @@ def test_log(self):
             ))
 
     def test_branch_log(self):
+        self.maxDiff = None
         with mocks.local.Git(self.path, git_svn=True):
             self.assertEqual(
                 run([
-                    local.Git.executable(), 'log', '--format=fuller', '--no-decorate', '--date=unix', 'branch-b...main',
+                    local.Git.executable(), 'log', '--format=fuller', '--no-decorate', '--date=unix', 'main..branch-b',
```

Comment:
Can you explain a bit why you reversed these?

Reply:
branch-b...main means branch-b U main - branch-b n main, so, it should include all commits from branch point of branch-b to head of branch-b and all commits form branch-b branch-point to head of main. The test purpose is not that. See all the commits in the brackets.

```
1@main--(2@main--3@main--4@main--5@main)
           |-(2.1@branch-a--2.2@branch-b--2.3@branch-b)
```

main..branch-b means branch-b U main - main, it should only include  all commits from branch point of branch-b to head of branch-b. See all the commits in the brackets.

```
1@main--2@main--3@main--4@main--5@main
           |-(2.1@branch-a--2.2@branch-b--2.3@branch-b)
```

Reply:
See more https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#Commit-Ranges